DateToolsSwift/DateTools/Date+TimeAgo.swift

```diff
@@ -112,12 +112,17 @@ public extension Date {
         else if (components.day! >= 2) {
             return self.logicalLocalizedStringFromFormat(format: "%%d %@days ago", value: components.day!)
         }
-        else if (isYesterday) {
-            if (numericDates) {
-                return DateToolsLocalizedStrings("1 day ago");
+        else if (components.day! >= 1) {
```

Comment:
If it is within 24 hours, `components.day` will be 0 so I think we need to include 0.
```suggestion
        else if ([0, 1].contains(components.day!)) {
```